app/controllers/LibAll.js

```diff
@@ -21,15 +21,20 @@ function renderAllLibrary(lib) {
       feedTemplate.renderFeedAsync(posts, options, callback);
     }
 
-    postModel.fetchPosts(
-      { repost: { $exists: false } },
-      null,
-      { after: options.after, before: options.before },
-      process,
-    );
+    const params = {
+      after: options.after,
+      before: options.before,
+    };
+    if (options.limit) params.limit = options.limit;
+
+    postModel.fetchPosts({ repost: { $exists: false } }, null, params, process);
```

Comment:
in order to stay closer to the initial implem (i.e. prevent the creation another constant), can you do this instead:

```suggestion

    postModel.fetchPosts(
      { repost: { $exists: false } },
      null,
      {
        after: options.after,
        before: options.before,
        ...(options.limit ? { limit: options.limit } : {}),
      },
      process,
    );
```